### DISCUSSION TITLE
Custom Module Development

### DISCUSSION THREAD

**Discussion by techravi007** [oqtane/oqtane.docs#45] [Q&A]
Can someone provide me reference for updated documentation / video-series  on custom module development ? 
This seems to be obsolete... [Getting Started with Oqtane Module Development](https://www.youtube.com/watch?v=JPfUZPlRRCE&list=PLYhXmd7yV0elLNLfQwZBUlM7ZSMYPTZ_f)

**Comment by thabaum** [comment]
@techravi007 Welcome to the Oqtane.Docs repository! :rocket: 

We would love to be sure everything is included in our documentation to help everyone get started with custom module development.  If you can provide details on where the documentation potentially falls short in your journey and may need updated we would love to hear what you have to share so we can make improvements.  

This is 100% community powered open-source projects and a few of us have gotten on board here at the Oqtane.Docs repository to improve things as we can while going down the road.  If you are able to help generate documentation pull requests or issues explaining each place that needs enhanced with clarification directed towards the latest version are welcome.  Then this can help us all identify how things can be written both here and in blogs and videos that could be generated by the community down the road.

I believe we should include a developer guide section located in here: https://docs.oqtane.org/guides/index.html

Not sure 100% how this all will be acceptable as I believe we are all flying by the seat of our pants in different directions currently. I will try to toss out an idea or two myself to help give a guide for us all.  We have some pretty extensive set of updates coming in Oqtane 5.1, we really need to catch up on 5.0.2 asap.  

I know @iJungleboy will be able to maybe help create something for a few areas in Concepts along with various support articles from a few other community members that helped contribute to the latest features in the Oqtane Framework.  At the same time things may get re-categorized a little if needed.  

The entire host and admin administration dashboards need updated to reflect all the latest.  

We just recently made a big push, and another is on it's way.  A lot of things were fixed as issues from updates in docfx so it now builds easy with a lot better support documentation on how to get docfx up and going, and how to contribute.

Any feedback on issues created and pull requests is very welcome, please share your experiences so we can all work together to help improve things for the next.  I currently have taken on a pretty massive task I am hoping to work on next week with installation setup.  I would love for anyone getting stuck while using them let us know so we can be sure to make note of things that can help the next person get stuck out the mud potentially.

If you navigate between docs, the Oqtane blog and review projects like the Oqtane Blog you should get an idea of how things can be structured, get them to load in your site, and all the different issues and expected behaviors that go along with.

Look forward to hearing more.

Cheers!
